src/runtime/windows_d3d12compute_x86.cpp

```diff
@@ -0,0 +1 @@
+#include "d3d12compute.cpp"
```

Comment:
#include'ing a .cpp file is unusual enough that a comment explaining it is appropriate.